cpu/cc2538/radio/cc2538_rf.c

```diff
@@ -119,20 +128,35 @@ void cc2538_init(void)
 
         NVIC_SetPriority(RF_ERR_ALT_IRQn, RADIO_IRQ_PRIO);
         NVIC_EnableIRQ(RF_ERR_ALT_IRQn);
+
+        NVIC_SetPriority(MAC_TIMER_ALT_IRQn, RADIO_IRQ_PRIO);
+        NVIC_EnableIRQ(MAC_TIMER_ALT_IRQn);
     }
     else {
         NVIC_SetPriority(RF_RXTX_IRQn, RADIO_IRQ_PRIO);
         NVIC_EnableIRQ(RF_RXTX_IRQn);
 
         NVIC_SetPriority(RF_ERR_IRQn, RADIO_IRQ_PRIO);
         NVIC_EnableIRQ(RF_ERR_IRQn);
+
+        NVIC_SetPriority(MACTIMER_IRQn, RADIO_IRQ_PRIO);
+        NVIC_EnableIRQ(MACTIMER_IRQn);
     }
 
+    RFCORE_SFR_MTMSEL &= ~CC2538_SFR_MTMSEL_MASK;
+    /* Select timer period */
+    RFCORE_SFR_MTMSEL |= CC2538_SFR_MTMSEL_TIMER_P;
+
+    /* Fix timer to Backoff period */
+    RFCORE_SFR_MTM0 |= TIMER_PERIOD_LSB;
+    RFCORE_SFR_MTM1 |= TIMER_PERIOD_MSB;
+
+    RFCORE_SFR_MTMSEL &= ~CC2538_SFR_MTMSEL_MASK;
```

Comment:
Yes, but because I had to set the 3 LSBs of RFCORE_SFR_MTMSEL to 001 (hence 146 and 148). For the next operation, I have to set those bits to 000 (so I have to clear it again).
If I do it the other way around the timer will start with the wrong configuration.

Reply:
should I mark is as resolved?

Reply:
Better add a comment.